src/utils/SaveData.java

```diff
@@ -0,0 +1,36 @@
+package utils;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class SaveData {
+
+    /**
+     * Save the data to a file
```

Comment:
Tu peux ajouter les paramètres dans le commentaire, genre

@param name The name of the password (voir par exemple dans Lfsr)